src/OrchardCore.Modules/OrchardCore.Widgets/ContentCardShapes.cs

```diff
@@ -0,0 +1,134 @@
+                            // ContentCard_Edit__[ParentContentType]__[CollectionType]__[ContentType] e.g. ContentCard_Edit__LandingPage__FlowPart__Service,
+                            // ContentCard_Edit__LandingPage__BagPart__Service, ContentCard_Edit__Form__FlowPart_Label
+                            contentCardEditor.Metadata.Alternates.Add($"{CONTENT_CARD_EDIT}__{parentContentType}__{collectionType}__{contentType}");
+                                                      
+                            if (!string.IsNullOrWhiteSpace(namedPart) && !(namedPart.Equals(collectionType) ))
+                            {
+                                //Define edit card shape for selected child  with specific type and partname per parent content type
+                                // ContentCard_Edit__[ParentContentType]__[PartName] e.g. ContentCard_Edit__Grid_LeftColumn, ContentCard_Edit__LandingPage__Services
+                                contentCardEditor.Metadata.Alternates.Add($"{CONTENT_CARD_EDIT}__{parentContentType}__{namedPart}");
+
+                                //Define edit card shape for selected child  with specific type and partname per parent content type
+                                // ContentCard_Edit__[ContentType]__[ContentType] e.g. ContentCard_Edit__Grid_LeftColumn_Client, ContentCard_Edit__LandingPage__Services__Service
+                                contentCardEditor.Metadata.Alternates.Add($"{CONTENT_CARD_EDIT}__{parentContentType}__{namedPart}__{contentType}");
+                           
+                            }
+                        }
+                    }
+                });
+
+            builder.Describe(CONTENT_CARD_FRAME)
+                .OnDisplaying(context =>
+                {
+                    // Alternates for Outer Frame of ContentCard
+                    dynamic contentCardFrame = context.Shape;
+                    string collectionType = context.Shape.ChildContent.CollectionShapeType;
+                    string contentType = context.Shape.ChildContent.ContentTypeValue as string;
+                    string parentContentType = context.Shape.ChildContent.ParentContentType;
+                    string namedPart = context.Shape.ChildContent.CollectionPartName;
+
+                    //Define Frame card shape per collection type
+                    // ContentCard_Frame__[CollectionType] e.g. ContentCard_Frame__FlowPart, ContentCard_Frame__BagPart, ContentCard_Frame__WidgetsListPart
+                    contentCardFrame.Metadata.Alternates.Add($"{CONTENT_CARD_FRAME}__{collectionType}");
+
+                    //Define Frame card shape per content type 
+                    // ContentCard_Frame__[ContentType] e.g. ContentCard_Frame__Paragraph, ContentCard_Frame__Form, ContentCard_Frame__Input
+                    contentCardFrame.Metadata.Alternates.Add($"{CONTENT_CARD_FRAME}__{collectionType}__{contentType}");
+
```

Comment:
the comment doesn't seem to match?

Reply:
Added missing alternate for content type